etc/gohan.yaml

```diff
@@ -67,4 +67,13 @@ extension:
   use:
   - gohanscript
  # - javascript
-  - go
\ No newline at end of file
+  - go
+
+  # default time limit in secs for running extension
+  timelimit: 10
+
+  # time limit overrides in secs for running particular extension in path
+  timelimits:
+  - path: /v1.0/store/orders
+    event: pre_*
+    timelimit: 1
```

Comment:
without looking through the source code it's hard to guess what is the unit here. timelimit_seconds?